hnn_core/network.py

```diff
@@ -788,15 +819,19 @@ def _attach_drive(self, name, drive, weights_ampa, weights_nmda, location,
                 src_idx = src_idx_end
                 for receptor in weights_by_type[target_cell_type]:
                     weights = weights_by_type[target_cell_type][receptor]
-                    self.add_connection(src_gids, target_gids_nested,
-                                        location, receptor, weights, delays,
-                                        space_constant)
+                    self.add_connection(
+                        src_gids=src_gids, target_gids=target_gids_nested,
+                        loc=location, receptor=receptor, weight=weights,
+                        delay=delays, lamtha=space_constant,
+                        probability=probability, conn_seed=drive['conn_seed'])
```

Comment:
```suggestion
                        probability=probability, seed=drive['conn_seed'])
```

I think here it's obvious that the seed is related to connectivity

Reply:
Totally agree, but I prefer to have named arguments stay the same when passed to multiple functions. Not a big deal to change if you disagree for this situation though.